site/app/controllers/student/SubmissionController.php

```diff
@@ -124,13 +124,22 @@ public function showHomeworkPage($gradeable_id, $gradeable_version = null) {
                     $can_inquiry = $this->core->getAccess()->canI("grading.electronic.grade_inquiry", ['graded_gradeable' => $graded_gradeable]);
                 }
 
+                $id = $this->core->getUser()->getId();
+
+                if($gradeable->isTeamAssignment()){
+                    $teamID = $this->core->getTeamIds($id, $gradeable_id);
```

Comment:
Put the actual query here. Something like
`$id = $this->core->getQueries()->getTeamByGradeableAndUser($gradeable_id, $id)->getId();`
Didn't look exactly into if that's exactly what you need, but you get the idea.